W-20116220-mulesoft-vibes-before-you-begin-fa (#696)
mulesoft/docs-code-builder: code-builder-home/modules/ROOT/pages/mulesoft-vibes.adoc

* Ensure you have the following permissions:  
** *Anypoint Code Builder Developer* (to use MuleSoft Vibes in the Cloud IDE)  
** *Mule Developer Generative AI User*  
* Make sure Einstein is enabled in Access Management. For more information, see xref:access-management::enabling-einstein.adoc[].
* Connect your Anypoint Platform organization to a Salesforce organization that has generative AI enabled. For setup instructions, see xref:access-management::connecting-salesforce-orgs.adoc[] and xref:access-management::trusted-salesforce-org.adoc[].
+

> NOTE: If you are a MuleSoft customer without an existing Salesforce organization, contact your MuleSoft account team to provision an Einstein-enabled Salesforce organization.

Reviewer: ```suggestion
NOTE: If you're a MuleSoft customer without an existing Salesforce organization, contact your MuleSoft account team to provision an Einstein-enabled Salesforce organization.
```
